src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/neu/NeuReforgeStoneJson.kt

```diff
@@ -0,0 +1,67 @@
+package at.hannibal2.skyhanni.data.jsonobjects.repo.neu;
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class NeuReforgeStoneJson(
+
+    @Expose
+    val internalName: String,
+
+    @Expose
+    val reforgeName: String,
+
+    @Expose
+    @SerializedName("itemTypes")
+    val rawItemTypes: Any,
```

Comment:
I don't particullarly like the idea of any but if it can't be changed its fine to keep. I dont think this is used in neu/skyhanni so it would be fine to change the format in neu repo to maybe have itemTypes lead to a map of itemTypes -> list of types and internalNames -> list of internal names. If wanted for extra precaution for not giving users with broken repos NPEs you could change "itemTypes" in neu repo to something else e.g. rawItemTypes.

This same logic can be applied to reforgeAbility where instead of having it be either a string or map of rarities to abilities you could have it map a general ability and then a per rarity one. But i think there is a better way to do this like maybe with a custom TypeAdapter like i do in #1146